### DISCUSSION TITLE
backup search engines, cookies and installed addons

### DISCUSSION THREAD

**Discussion by eadmaster** [floccusaddon/floccus#1213] [Ideas]
Provide options backup search engines, cookies and installed addons.

Would be useful with chome-based browsers that [cannot use Google Sync anymore since March 15 2021](https://www.omgubuntu.co.uk/2021/01/chromium-sync-google-api-removed).

**Comment by marcelklehr** [comment]
Hey @eadmaster 

Thank you for your feedback. I'm sorry to say that syncing additional items is out of scope for this app. I understand the need for more, but this app is quite a lot of work as it is already.
